requirements.txt

```diff
@@ -3,7 +3,7 @@
 numpy>=1.15.0
 pandas>=0.25.1
 keras>=2.2.4
-tensorflow>=1.14.0
+tensorflow==1.14.0
```

Comment:
När jag drar in tensorflow 2.0 på Ubuntu 18.04 kraschar PolicyGradient-agenterna vid anropet tf.log(...). Se https://www.google.com/search?q=tf.log+is+deprecated. Detta och mycket annat behöver fixas om vi väljer att gå över till tf 2.0.

Reply:
Vi bör uppgradera till TF 2.0. Ska bara vara säkra på att vi kan kör 2.0 på hetzner. Det borde inte vara ett problem men det är bäst att verifiera först.